pandas/tests/indexes/test_numeric.py

```diff
@@ -114,6 +114,22 @@ def test_numeric_compat(self):
         result = idx**2.0
         tm.assert_index_equal(result, expected)
 
+    @pytest.mark.xfail(reason="Index._add_numeric_methods_binary does not "
```

Comment:
is there an issue number ? I don't actually like adding a lot of xfails, if its only 1 ok

Reply:
Yah, 19042.  I'm removing this test.